src/applications/common/schemaform/save-in-progress/RoutedSavableReviewPage.jsx

```diff
@@ -103,18 +118,25 @@ class RoutedSavableReviewPage extends React.Component {
       user
     } = this.props;
 
+    const downtimeDependencies = get('downtime.dependencies', formConfig) || [];
+
     return (
       <div>
         <ReviewChapters
           formConfig={formConfig}
           formContext={formContext}
           pageList={pageList}
           onSetData={() => this.debouncedAutoSave()}/>
-        <SubmitController
-          formConfig={formConfig}
-          pageList={pageList}
-          path={path}
-          renderErrorMessage={this.renderErrorMessage}/>
+        <DowntimeNotification
+          appTitle="application"
+          render={this.renderDowntime}
+          dependencies={downtimeDependencies}>
+          <SubmitController
+            formConfig={formConfig}
+            pageList={pageList}
+            path={path}
+            renderErrorMessage={this.renderErrorMessage}/>
+        </DowntimeNotification>
         <SaveStatus
```

Comment:
Does wrapping SubmitController disable just the submit button?

Reply:
That and a back button, but I'm not too concerned about that since you can edit all of your info on the review page (plus use the browser back button).